Source/WebKit/UIProcess/API/Cocoa/WKWebViewConfiguration.mm

```diff
@@ -297,6 +300,11 @@ - (void)encodeWithCoder:(NSCoder *)coder
     [coder encodeObject:self.userContentController forKey:@"userContentController"];
     [coder encodeObject:self.websiteDataStore forKey:@"websiteDataStore"];
 
+#if ENABLE(WK_WEB_EXTENSIONS)
+    if (auto *controller = _webExtensionController.peek())
+        [coder encodeObject:controller forKey:@"webExtensionController"];
```

Comment:
It looks like we don't put these keys in constants anywhere else.

Reply:
(I was going to say maybe we should, but it wouldn't be consistent)

Reply:
Yeah, less of that in WebKit.